src/main/resources/META-INF/rewrite/ibm-java.yml

```diff
@@ -110,4 +112,38 @@ recipeList:
   - org.openrewrite.java.ChangeType:
       oldFullyQualifiedTypeName: com.ibm.security.auth.module.Krb5LoginModule
       newFullyQualifiedTypeName: com.sun.security.auth.module.Krb5LoginModule
-      ignoreDefinition: true
\ No newline at end of file
+      ignoreDefinition: true
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.RemovedJavaXMLWSModuleProvided
+displayName: Do not use `java.xml.ws` module in application
+description: The `java.xml.ws` module was removed in Java11.
```

Comment:
Let's update the description to something like this:
```The `java.xml.ws` module was removed in Java11. Websphere Liberty provides its own implementation of the module, which can be used by specifying the `jaxws-2.2` feature in the server.xml file. This recipe removes the javax.xml.ws module from the applications builds dependency in favor of the Websphere Liberty implementation to avoid class loading issues.```

Reply:
updated in next commit